resources/views/auth/register.php

```diff
@@ -44,9 +44,15 @@ class="logo__image">
                 <dt-button context="success"
                            type="submit"
                            class="button--large button--block">
-                    <?php esc_html_e('Register', 'disciple-tools-autolink') ?>
+                    <?php esc_html_e('Register', 'dt-launcher') ?>
```

Comment:
@incraigulous , make sure the dt/plugin doesn't say "disciple-tools-autolink" here